### PR TITLE
CA-174979: Confusing right-click add to pool menu (if you accept the defaults)

### PR DESCRIPTION
**Pull request by GaborApatiNagy**
Changed "New Pool" to "Unnamed Pool"

Signed-off-by: Gabor Apati-Nagy <gabor.apati-nagy@citrix.com>

### CODE REVIEW

**Comment by stephen-turner** [comment]
I want to see this one please. I don't think I agree with the suggestion in the ticket, but I want to see it to be sure.

**Comment by kc284** [comment]
Personally I don't like "unnamed pool" very much. How about offering a name like "pool (1)", like we do for the new VMs? (I've put this comment on the ticket too).

**Comment by GaborApatiNagy** [comment]
I changed it to say "Pool"

**Comment by stephen-turner** [comment]
I still want to see a demo of this.

**Comment by stephen-turner** [comment]
I don't really like calling the pool "Pool" either. I propose leaving the name blank so the the user is forced to fill in a name before continuing.

**Comment by stephen-turner** [comment]
I also think we should implement Tina's separate suggestion in the same ticket, if it's easy.

File: XenModel/Messages.resx [hunk 1]
```diff
@@ -8461,7 +8461,7 @@ Once the VM has restarted click the Install [XenServer product] Tools menu item
     <value>&amp;New Policy...</value>
   </data>
   <data name="NEW_POOL" xml:space="preserve">
```
Comment (kc284) [inline XenModel/Messages.resx]: I believe this string is not used anywhere any more - can we remove it to keep the code lean?

Comment (GaborApatiNagy) [inline XenModel/Messages.resx]: Makes sense, removed it now

**Comment by kc284** [comment]
Looks ok now; merging.

**Pull request merged**